### ISSUE TITLE
Auto-entities filter doesn't work with fold-entity-row + dummy-entity-row.

### ISSUE THREAD

**Issue by jazzyisj** [thomasloven/lovelace-dummy-entity-row#2]
Hassio 0.91.3, HassOS
Cards are all updated to latest version with custom updater.

I have an entities card that contains a fold-entity row which in turn contains an auto-entities filter that displays devices on my network which have a state of Home.   The filter works fine if the head type specified in the fold-entity-row: is section. 

```
          - type: custom:fold-entity-row
            head:
              type: section
              label: Network Devices
            entities:
            - type: custom:auto-entities
              filter:
                include:
                  - group: group.all_devices
                    options:
                      secondary_info: last-changed
                exclude:
                  - state: Away
                  - state: unknown
                  - state: not_home
                  - domain: person
              card:
                type: entities
```

![image](https://user-images.githubusercontent.com/44064559/56084399-9c7d9e80-5e00-11e9-80ed-75fa5cacfdfb.png)

If a use a dummy-entity-row for the head type in the fold-entity-row the filter does not work and the secondary info disappears.  I can reinstate the last-changed info by adding it to the fold-entity configuration.

```
          - type: custom:fold-entity-row
            head:
              **type: custom:dummy-entity-row**
              entity: group.all_devices
              icon: mdi:network
              name: Network Devices
            **group_config:**
              **secondary_info: last-changed**
            entities:
            - type: custom:auto-entities
              filter:
                include:
                  - group: group.all_devices
                    options:
                      secondary_info: last-changed
                exclude:
                  - state: Away
                  - state: unknown
                  - state: not_home
                  - domain: person
              card:
                type: entities
```

![image](https://user-images.githubusercontent.com/44064559/56084374-3bee6180-5e00-11e9-8468-2babdeb9e1f6.png)

**Comment by thomasloven** [comment]
Did you try the recommended way of combining auto-entities with fold-entity-row presented here: https://github.com/thomasloven/lovelace-auto-entities#about-monster-card ?

**Comment by thomasloven** [comment]
No reply for seven weeks.
Closing

**Comment by jazzyisj** [comment]
@thomasloven 

My apologies for the extremely long delay in replying.  My rPI died back in May and I decided to take the summer off to tend to my gardens and it took me a while to get my new rPI up and running and then wrap my head around all the recent changes in HA.

Anyway, no I obviously did not try the recommended way or I would not have needlessly wasted your time here.  I guess I didn't read far enough down in the readme.  I promise I'll do better next time!  I have now tried the recommended way and it works perfectly.

Thank you.
